site/academics/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Your Website Name</title>
+  <link rel="stylesheet" href="/assets/css/reset.css" type="text/css" />
+  <link rel="stylesheet" href="/assets/css/global.css" type="text/css" />
+  <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <h1>Academics</h1>
+  <p>Jodie Mcsally Dance Program, Class of 21</p>
+  <div>
+    <div>Majors</div>
+    <div>Inquiry & innovation</div>
+  </div>
+  <div>Support for IEP</div>
+  <div>Lorem Ipsum</div>
+  <div>Learn more</div>
```

Comment:
Good start @JeremiahHarris19. Next step I would go ahead and do would be to start adding HTML to group related content together. For example here you have:

```
<div>Support for IEP</div>
<div>Lorem Ipsum</div>
<div>Learn more</div>
<div> Dual Curriculum</div>
<div>Lorem Ipsum</div>
<div>Learn More</div>
```

I'd recommend starting to add HTML that groups each one of those blocks together, like so:

```
<div>
  <div>Support for IEP</div>
  <div>Lorem Ipsum</div>
  <div>Learn more</div>
</div>
<div>
  <div> Dual Curriculum</div>
  <div>Lorem Ipsum</div>
  <div>Learn More</div>
</div>
```

from there you could add classes to the html, like so:

```
<div class="learn-more-block">
  <div class="learn-more-title">Support for IEP</div>
  <div class="learn-more-text">Lorem Ipsum</div>
  <div class="learn-more-button">Learn more</div>
</div>
<div class="learn-more-block">
  <div class="learn-more-title">Dual Curriculum</div>
  <div class="learn-more-text">Lorem Ipsum</div>
  <div class="learn-more-button">Learn More</div>
</div>
```

then you would use those classes in CSS to style them.